app/src/main/res/layout/view_address.xml

```diff
@@ -14,7 +14,6 @@
         android:paddingTop="2dp"
         android:paddingBottom="3dp"
         android:background="@drawable/address_background_themed"
-        app:layout_constraintHorizontal_bias="1"
```

Comment:
By removing this line it will break right aligned addressviews in TransactionInfo dialog